### PR TITLE
What's New in SwiftWasm #3

### PR DESCRIPTION
**Pull request by MaxDesiatov**
I hope I didn't miss anything, suggestions and feedback are welcome as usual.

I've also added new fonts to the theme so that it feels less bland.

Before:

Screenshot 2020-10-29 at 22 09 38

After:

Screenshot 2020-10-29 at 22 09 50

### CODE REVIEW

File: Content/posts/update-03-october-2020.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+and is now available via Homebrew on macOS.
+
+After the release, [@carson-katri](https://github.com/carson-katri) merged a PR that greatly improves
+command-line experience with `carton test`. It builds on his previous work on `carton dev` with
+compiler diagnostics parsing and error highlighting. With this new change XCTest output
+is also parsed, reformatted, and highlighted in a nice summary view.
+
+<br />
+
+![Pretty-printed test summary in `carton`](/images/update-03-carton-test-summary.png width=100%)
```
Comment (kateinoigakukun) [inline Content/posts/update-03-october-2020.md]:
```suggestion
![Pretty-printed test summary in `carton`](./images/update-03-carton-test-summary.png width=100%)
```

File: Content/posts/update-03-october-2020.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+<br />
+<br />
+
+![Pretty-printed test errors in `carton`](/images/update-03-carton-test-errors.png width=100%)
```
Comment (kateinoigakukun) [inline Content/posts/update-03-october-2020.md]:
```suggestion
![Pretty-printed test errors in `carton`](./images/update-03-carton-test-errors.png width=100%)
```

**Review by kateinoigakukun — APPROVED** [review]

File: Content/posts/update-03-october-2020.md [hunk 3]
```diff
@@ -12,7 +12,22 @@ his experience when integrating SwiftWasm apps within Go apps through [Wasmer](h
...
-integrate binaries produced by SwiftWasm in any host application.
+integrate binaries produced by SwiftWasm in a host application.
+
+[@johngarrett](https://github.com/johngarrett) published an interactive ["Swift, Wasm, and
+Algorithms"](https://garrepi.dev/swal) documentation page. It is a demo of the recently released
+[Swift Algorithms](https://github.com/apple/swift-algorithms) package where you can tweak parameters
+for some of the functions and observe results directly in the browser. Not all of the functions are
+fully interactive now, but it's a great proof of concept for interactive documentation websites one
+could build with SwiftWasm. Its source code is [available on
+GitHub](https://github.com/apple/swift-algorithms).
```
Comment (yonihemi) [inline Content/posts/update-03-october-2020.md]:
```suggestion
GitHub](https://github.com/johngarrett/swal-wasm).
```

Comment (MaxDesiatov) [reply Content/posts/update-03-october-2020.md]: Great catch, thanks!

**Review by kateinoigakukun — APPROVED** [review]

**Review by yonihemi — APPROVED** [review]

File: Content/posts/update-03-october-2020.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+## Upstream PRs
+
+### Swift Driver
+
+[@owenv](https://github.com/owenv) merged [a PR to the Swift Driver
+project](https://github.com/apple/swift-driver/pull/315) implementing WebAssembly toolchain
+support. This mirrors our existing [C++ implementation in the legacy
+driver](https://github.com/swiftwasm/swift/blob/swiftwasm/lib/Driver/WebAssemblyToolChains.cpp), and
+it's great that the new parts of the Swift compiler rewritten in Swift are going to support
+WebAssembly too. While Swift Driver in that enabled in any toolchain by default, we're definitely
```
Comment (carson-katri) [inline Content/posts/update-03-october-2020.md:121]: I'm not quite sure what the beginning of this sentence is saying. Maybe:
```suggestion
WebAssembly too. While Swift Driver isn't enabled in any toolchain by default, we're definitely
```

Comment (MaxDesiatov) [reply Content/posts/update-03-october-2020.md:121]: Thanks, that's updated in d02849836d88bd744f89b21420920409ae563151.

**Pull request merged**